content/en/blog/runtime-security.md

```diff
@@ -0,0 +1,276 @@
+02:58:44.979413526: Notice A shell was spawned in a container with an attached terminal (user=root k8s.ns=flag k8s.pod=nginx-7cdbd8cdc9-67gv6 container=268a28e93f14 shell=bash parent=docker-runc cmdline=bash terminal=34816 container_id=268a28e93f14 image=nginx) k8s.ns=flag k8s.pod=nginx-7cdbd8cdc9-67gv6 container=268a28e93f14 k8s.ns=flag k8s.pod=nginx-7cdbd8cdc9-67gv6 container=268a28e93f14
+02:49:28.743307008: Warning Pod started with privileged container (user=system:serviceaccount:default:nova-sa pod=entry ns=default image=nginx)
+02:52:41.920078080: Warning Disallowed namespace created (user=system:unsecured ns=my-new-namespace)
+02:54:26.914213120: Warning Pod started with privileged container (user=system:serviceaccount:default:nova-sa pod=entry ns=default image=nginx)
+02:54:31.269857024: Notice Attach/Exec to pod (user=system:serviceaccount:default:nova-sa pod=entry ns=default action=attach command=<NA>)
+02:56:45.194068725: Notice A shell was spawned in a container with an attached terminal (user=root k8s.ns=falco k8s.pod=falco-ds-zdsgm container=493b88ce97ff shell=bash parent=docker-runc cmdline=bash terminal=34816 container_id=493b88ce97ff image=falcosecurity/falco) k8s.ns=falco k8s.pod=falco-ds-zdsgm container=493b88ce97ff k8s.ns=falco k8s.pod=falco-ds-zdsgm container=493b88ce97ff
+02:56:50.233962101: Warning Shell history had been deleted or renamed (user=root type=openat command=bash fd.name=/root/.bash_history name=/root/.bash_history path=<NA> oldpath=<NA> k8s.ns=falco k8s.pod=falco-ds-zdsgm container=493b88ce97ff) k8s.ns=falco k8s.pod=falco-ds-zdsgm container=493b88ce97ff k8s.ns=falco k8s.pod=falco-ds-zdsgm container=493b88ce97ff
+02:57:31.939322303: Notice A shell was spawned in a container with an attached terminal (user=root k8s.ns=falco k8s.pod=falco-ds-zdsgm container=493b88ce97ff shell=bash parent=docker-runc cmdline=bash terminal=34816 container_id=493b88ce97ff image=falcosecurity/falco) k8s.ns=falco k8s.pod=falco-ds-zdsgm container=493b88ce97ff k8s.ns=falco k8s.pod=falco-ds-zdsgm container=493b88ce97ff
+02:57:31.893552896: Notice Attach/Exec to pod (user=system:unsecured pod=falco-ds-zdsgm ns=falco action=exec command=%2Fbin%2Fbash)
+02:58:44.943840000: Notice Attach/Exec to pod (user=system:unsecured pod=nginx-7cdbd8cdc9-67gv6 ns=flag action=exec command=%2Fbin%2Fbash)
+```
+
+It is important to note that Falco currently is not running on the master node. Furthermore Falco would alerts on itself as it needs to run in privileged mode to access the kernel events. 
+
+# Findings
+
+We were able to bypass Kubernetes RBAC by using a known exploit. We were then able to disable the policy enforcement and create a new namespace. Furthermore we were able to list namespaces and thus find the flag. 
```

Comment:
This `We were then able to disable the policy enforcement` feels like it should read `We were then able to disable policy enforcement`. Your call

Reply:
Also a comma after `Furthermore`